samples/tictactoe/common/src/main/java/com/squareup/sample/gameworkflow/RunGameWorkflow.kt

```diff
@@ -42,6 +42,7 @@ typealias RunGameScreen = AlertContainerScreen<PanelContainerScreen<Any, Any>>
  * We define this otherwise redundant typealias to keep composite workflows
  * that build on [RunGameWorkflow] decoupled from it, for ease of testing.
  */
+@OptIn(WorkflowUiExperimentalApi::class)
```

Comment:
I don't see why this is needed, possible b/c of the `@WorkflowUiExperimentalApi` mistake above? Same question for the other new appearances.

Reply:
The compiler is more strict. It's needed even with the other change.
```
> Task :samples:tictactoe:common:compileKotlin FAILED
e: /Users/ralf/Development/projects/square/workflow-kotlin/samples/tictactoe/common/src/main/java/com/squareup/sample/gameworkflow/RunGameWorkflow.kt: (46, 59): This declaration is experimental and its usage must be marked with '@com.squareup.workflow1.ui.WorkflowUiExperimentalApi' or '@OptIn(com.squareup.workflow1.ui.WorkflowUiExperimentalApi::class)'
```